src/lib/reasoners/satml.ml

```diff
@@ -999,9 +1004,11 @@ module Make (Th : Theory.S) : SAT_ML with type th = Th.t = struct
   let print_aux fmt hc =
     Format.fprintf fmt "%a@," Atom.pr_clause hc
 
+  let is_unsat env : unit = env.is_unsat <- true
```

Comment:
I know we have the confusing `is_mine` already but let's not add functions called `is_XXX` that do anything else than return whether their argument satisfies the "XXX" property. If this function is really needed, call it `set_unsat` (but I find `env.is_unsat <- true` to be more clear; having a function for that kind of implies that there is something more going on).